packages/docs/src/components/layout/FilterDialog/FilterDialog.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useRef } from 'react';
+import NativeDialog from '@cmsgov/design-system/src/components/NativeDialog/NativeDialog.tsx';
+import uniqueId from 'lodash/uniqueId';
+import classNames from 'classnames';
+import mergeRefs from '@cmsgov/design-system/src/components/utilities/mergeRefs.ts';
+import CloseButton from './CloseButton';
+
+export interface FilterDialogProps {
+  /**
+   * Buttons or other HTML to be rendered in the "actions" bar at the bottom of
+   * the dialog. Should include a button for applying the user's selections and
+   * one for closing the dialog.
+   */
+  actions: React.ReactNode;
+  /**
+   * The main dialog content
+   */
+  children: React.ReactNode;
+  /**
+   * Additional classes to be added to the root dialog element.
+   */
+  className?: string;
+  /**
+   * Text for the FilterDialog heading. Required because the `heading` will be focused on mount.
+   */
+  heading: string | React.ReactNode;
+  /**
+   * A unique `id` to be used on heading element to label multiple instances of FilterDialog.
+   */
+  headingId?: string;
+  /**
+   * Heading type to override default `<h3>`
+   */
+  headingLevel?: '1' | '2' | '3' | '4' | '5';
```

Comment:
why don't we ever include level 6 in our heading levels?

Reply:
heh, this is just copypasta from the dialog. Maybe because that seems excessive?